src/Symfony/Bundle/TwigBundle/Resources/config/twig.xml

```diff
@@ -71,6 +71,18 @@
             <argument type="service" id="templating.locator" />
         </service>
 
+        <service id="twig.extension.profiler" class="Symfony\Bridge\Twig\Extension\ProfilerExtension" public="false">
+            <argument type="service" id="twig.profile" />
+            <argument type="service" id="debug.stopwatch" on-invalid="ignore" />
+        </service>
+
+        <service id="twig.profile" class="Twig_Profiler_Profile" />
```

Comment:
I made public because you might want to get the profile from a CLI tool or something else. It is available through the data collector, but exposing it directly seems like a better idea.